native/cocos/editor-support/spine-creator-support/SkeletonRenderer.cpp

```diff
@@ -1041,3 +1042,82 @@ cc::Material *SkeletonRenderer::requestMaterial(uint16_t blendSrc, uint16_t blen
     }
     return _materialCaches[key];
 }
+
+void SkeletonRenderer::setSlotTexture(const std::string &slotName, cc::Texture2D* tex2d, bool createAttachment) {
+    if (!_skeleton) return;
+    auto slot = _skeleton->findSlot(slotName.c_str());
+    if (!slot) return;
+    auto attachment = slot->getAttachment();
+    if (!attachment) return;
+    auto width = tex2d->getWidth();
+    auto height = tex2d->getHeight();
+
+    if (createAttachment) {
+        attachment = attachment->copy();
```

Comment:
Here to create new attachment. But the above logic will return if attachment is null. The logic seems wired.